avm/res/document-db/database-account/sql-role/main.bicep

```diff
@@ -0,0 +1,66 @@
+metadata name = 'DocumentDB Database Account SQL Role.'
+metadata description = 'This module deploys SQL Role Definision and Assignment in a CosmosDB Account.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Database Account. Required if the template is used in a standalone deployment.')
+param databaseAccountName string
+
+@description('Required. Name of the SQL Role.')
+param name string
+
+@description('Optional. An array of data actions that are allowed.')
+param dataActions array = [
+  'Microsoft.DocumentDB/databaseAccounts/readMetadata'
```

Comment:
As i mentioned in my other comment. We need to either have defaults value here or in its children files but not in both as those defaults values are duplicated and generated extra noise and confusion